login/templates/account/email/email_confirmation_message.txt

```diff
@@ -0,0 +1,13 @@
+{% load account %}
```

Comment:
Estas instruções estão sendo usadas? Averiguei que não estão causando impacto algum nos emails. Sugiro a remoção de tudo que for desnecessário.

Reply:
Na minha opinião a sugestão não impacta diretamente no funcionamento da funcionalidade.